### ISSUE TITLE
Q: Error when running 

### ISSUE THREAD

**Issue by czfoo** [LiuLabUB/HMMRATAC#78]
Dear Sir/Madam,

When I tried to run HMMRATAC, I got this error message:

Exception in thread "main" net.sf.samtools.SAMFormatException: SAM validation error: ERROR: Record 1, Read name NS500138:357:HG7KCBGXB:3:23404:16812:5744, RG ID on SAMRecord not found in header: 

Do you by any chance know what this error is?

Thank you.

**Comment by EvanTarbell** [comment]
Did you trim the reads with DeepTools alignmentSieve? This is a common cause of this issue. If so, check out #47

**Comment by czfoo** [comment]
No, I did not trim them with alignmentSieve. But I trimmed the fastq files with Trimmomatic before aligning them. Should I re-try doing the alignment without trimming? Also, is Bowtie2 fine for aligning the reads, or should I use a different program? Or is there any special settings I need to run Bowtie2 on?
